### PR TITLE
Send mail at action point creation

### PR DESCRIPTION
**Pull request by simon-the-sloth**
fixes uncief/etools-issues#74

### CODE REVIEW

File: EquiTrack/t2f/models.py [hunk 1]
```diff
@@ -638,6 +640,45 @@ def save(self, *args, **kwargs):
 
         super(ActionPoint, self).save(*args, **kwargs)
 
+        if created:
+            self.send_notification_email()
+
+    def send_notification_email(self):
```
Comment (ntrncic) [inline EquiTrack/t2f/models.py]: could this be done as a post_save signal?

File: EquiTrack/t2f/models.py [hunk 2]
```diff
@@ -638,6 +640,45 @@ def save(self, *args, **kwargs):
...
+        html_content = render_to_string('emails/action_point_assigned.html', context)
+
+        # TODO what should be used?
+        sender = settings.DEFAULT_FROM_EMAIL
+        msg = EmailMultiAlternatives(subject, '',
+                                     sender, [recipient],
+                                     cc=[cc])
+        msg.attach_alternative(html_content, 'text/html')
+
+        # Core mailing is broken. Multiple headers will throw an exception
```
Comment (ntrncic) [inline EquiTrack/t2f/models.py]: please remove code

File: EquiTrack/t2f/models.py (hunk 1 above)
Comment (ntrncic) [inline EquiTrack/t2f/models.py]: can't we reuse the send_notficiation_email in the Travel model? Or move the whole function into a mixin?

File: EquiTrack/t2f/serializers/mailing.py [hunk 3]
```diff
@@ -33,3 +33,12 @@ class Meta:
...
                   'reference_number', 'purpose', 'rejection_note')
+
+
+class ActionPointMailSerializer(serializers.Serializer):
+    person_responsible = serializers.CharField(source='person_responsible.get_full_name')
+    assigned_by = serializers.CharField(source='assigned_by.get_full_name')
+    action_point_number = serializers.CharField()
+    travel_reference_number = serializers.CharField(source='travel.reference_number')
+    description = serializers.CharField()
+    due_date = serializers.DateTimeField(format='%m/%d/%Y %H:%M')
```
Comment (ntrncic) [inline EquiTrack/t2f/serializers/mailing.py]: is this format consistent with others in the app?

File: EquiTrack/t2f/models.py (hunk 2 above)
Comment (simon-the-sloth) [inline EquiTrack/t2f/models.py]: Done

File: EquiTrack/t2f/models.py (hunk 1 above)
Comment (simon-the-sloth) [inline EquiTrack/t2f/models.py]: Not really. The code is slightly different.

File: EquiTrack/t2f/serializers/mailing.py (hunk 3 above)
Comment (simon-the-sloth) [inline EquiTrack/t2f/serializers/mailing.py]: Yes

**Pull request merged**